### ISSUE TITLE
eventbus 1.7.8 causes issues

### ISSUE THREAD

**Issue by joecoolio66** [PrestaShop/PrestaShop#28870]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

We upgraded the version of eventbus to 1.7.8 yesterday morning and it immediately caused issues with our site generating 500 errors upon checkout.  We switched to debug mode and it looks like whatever overrides they added were clashing with our Canada Post and Caching modules.  

We had to revert back to 1.7.6 and kill the cache in var/cache before we were able to re-enable the Canada Post and Caching modules.  We are lucky we had a backup of our 1.7.8.5 previous site to copy over the version because eventbus doesn't appear to be part of the PS download, so we couldn't locate a previous version.

We are running 1.7.8.6

### Expected behavior

There should be no issues upon upgrade.  This is a regression

### Steps to reproduce

PS 1.7.8.6
eventbus 1.7.8

We assume it will cause issues with other modules that reach out to external resources.

### PrestaShop version(s) where the bug happened

1.7.8.6

### PHP version(s) where the bug happened

7.4.29

### If your bug is related to a module, specify its name and its version

ps_eventbus

**Comment by AureRita** [comment]
Hi @joecoolio66 , 

Thank you for submitting this issue. Unfortunately, there is not enough information provided to reproduce it and work on it. 
**You must follow the template and provide clear and concise steps to reproduce the bug**. 
Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/)

Currently, I tried to reproduce it with the few information I've so I try the checkout with the module and the good version of prestashop and I didn't have any issue, as you can see : 

https://user-images.githubusercontent.com/100699445/175976992-484c7fbe-0bdd-429c-833d-1896a9733f50.mp4

**Comment by joecoolio66** [comment]
We know... we were too busy trying to fix the issue and didn't take screenshots of the error trace.   It looked like main issue was clashing with the Canada Post module overrides.  Anyway, hopefully someone else will also report issues... as it stands right now, we can't use the latest version of the eventbus module.  

Just a bit more information - After upgrading, we found we couldn't access customer accounts... it would take forever and then cloudflare would give us a 523 timeout error.  It was the same issue with accessing front end product pages... it just tried to load and then timed out.  

If we tried to checkout, it gave a general 500 error and then every page got a 500 error until we killed the files in var/cache again.

All we can really tell you is 1.7.6 works great with our modules, but 1.7.8 doesn't.  The stack trace on the 500 error showed specifically our canada post and cache modules and overrides.
